provision-mysql.sh

```diff
@@ -49,7 +49,7 @@ configure_mysql()
 		tell_status "installing $_my_cnf"
 		tee -a "$_my_cnf" <<EO_MY_CNF
 [mysqld]
-innodb_doublewrite = off
+#innodb_doublewrite = off
```

Comment:
Thanks for the explanation and link. Having that stored here for posterity is likely to help folks in the future.